tests/check_framework/models.py

```diff
@@ -6,10 +6,14 @@ class SimpleModel(models.Model):
     field = models.IntegerField()
     manager = models.manager.Manager()
 
+class CheckModel(models.Model):
+    check_ran = models.BooleanField(default=False)
```

Comment:
```suggestion
    pass
```

You can actually drop this field, since the test is doing `.exists()` - the implicit `AutoField` is all that's needed to create the table.